Refleaks bot failure seems unrelated:

2:10:42 load avg: 5.18 [ 58/428/1] test_mailbox failed (1 failure) (2 hour 31 sec) -- running: test_gzip (1 hour 44 min), test_cmd_line (1 hour 39 min), test_argparse (1 hour 39 min)
beginning 6 repetitions
123456
..test test_mailbox failed -- Traceback (most recent call last):
  File "D:\buildarea\pull_request.ware-win81-release.refleak\build\lib\test\test_mailbox.py", line 411, in test_set_item
    self.assertEqual(len(self._box), 2)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 3 != 2

--- a/Lib/test/test_code.py
+++ b/Lib/test/test_code.py
@@ -212,7 +212,7 @@ class CodeTest(unittest.TestCase):
         CodeType = type(co)
 
         # test code constructor
-        return CodeType(co.co_argcount,
+        CodeType(co.co_argcount,
                         co.co_posonlyargcount,
                         co.co_kwonlyargcount,
                         co.co_nlocals,

